KarhooSDK/Network/HttpClient/TokenRefreshingHttpClient/Interactor/KarhooRefreshTokenInteractor.swift

```diff
@@ -10,16 +10,35 @@ import Foundation
 
 final class KarhooRefreshTokenInteractor: RefreshTokenInteractor {
 
+    // MARK: - Nested types
+
+    private enum Constants {
+        static let MaxTimeIntervalToRefreshToken = TimeInterval(30) // 30 sec
```

Comment:
Is this the same value as on Android?

Reply:
Value removed and replace with refreshBuffer :)